geomesa-kafka/geomesa-kafka-datastore/pom.xml

```diff
@@ -70,6 +79,13 @@
             <artifactId>kafka_2.10</artifactId>
         </dependency>
         <dependency>
+            <!--This is a compile dependency of Kafka and also KafkaGeoMessageConsoleConsumer -->
+            <groupId>net.sf.jopt-simple</groupId>
+            <artifactId>jopt-simple</artifactId>
+            <version>3.2</version>
+            <scope>compile</scope>
```

Comment:
1.  Do we have a IP CQ in for this dependency?  If not, I'll need some details.
2.  Let's try and manage the version in the root pom.